packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -25,7 +25,7 @@ export class OfficeMockObject {
   }
 
   /**
-   * Adds a mock object to OfficeMockObject. The object can be accessed by simply calling `this.objectName`
+   * addMock(name) will add a property named “name”, with a new OfficeMockObject as its value, to the object
    * @param objectName Object name of the object to be added
    */
   addMock(objectName: string) {
```

Comment:
What happens if this is called twice with the same name?

Reply:
The same as the new "set" function, it will simply override what is there.
Could we change to setMock as well? We would then have more consistent names

Reply:
If we're overwriting the value then 'set' makes more sense to me . . . but are we then generating confusion between what the difference between 'set' and 'setMock' is?  For that matter why 'addMock' and 'set'.  I don't think is easily clear why developers would use one or the other.

I suppose I should be bringing this up in the email thread instead.

Reply:
I suppose we can continue this PR now?

Reply:
`setMock` is preferable per above explanation.

Reply:
I believe the current thinking is to have 'set' which allows for setting the property or function of an Office object and 'addMock' which doesn't overwrite and adds a new mock office object.